tasks/common.js

```diff
@@ -233,7 +233,12 @@ module.exports = function (grunt) {
         return _platform;
     }
     
-    function arch() {
+    function arch(hint) {
+		hint = !hint && platform() === 'win' ? 'ia32' : hint;
```

Comment:
we can add platform() === 'win' || platform() === "linux " and then it will be possible to remove the code below added for linux.

Reply:
Done.